docker-compose.yml

```diff
@@ -42,8 +42,6 @@ services:
     << : *services-defaults
     command: ["python3", "-m", "pathfinding_service.cli", "--matrix-server", "https://transport.${SERVER_NAME}"]
     restart: always
-    ports:
-      - 6001:6000
```

Comment:
Why don't we need those anymore? Does traefik manage that automatically?

Reply:
The `ports` section is only needed when you want to expose a port on the host interface.
Since traefik is running inside the same docker network it can access the other containers directly.